website_and_docs/content/documentation/grid/advanced_features/endpoints.zh-cn.md

```diff
@@ -43,90 +43,66 @@ cURL --request DELETE 'http://localhost:4444/session/<session-id>'
 
 ### 删除节点
 
-要从Grid中删除节点, 
-请使用下面列出的cURL命令.
-它不会停止在该节点上运行的任何持续中的会话.
-除非显式终止, 否则节点将继续按原样运行.
-分发器不再知晓该节点,
-因此任何匹配的新会话请求
-也不会转发到该节点.
+要从网格中删除节点，请使用下面列出的 cURL 命令。该命令不会停止正在该节点上运行的任何会话。节点会继续运行，除非被明确杀死。分发器不再知道该节点，因此任何匹配的新会话请求都不会转发到该节点。
```

Comment:
Do we have the possible to translate as another style for "The Node continues running as it is unless explicitly killed" ?